templates/.github/workflows/prod-pipeline.yml

```diff
@@ -0,0 +1,76 @@
+# .github/workflows/prod-pipeline.yml
+# This is a github action pipeline for production.
+name: Production Deployment
+on: 
+  workflow_dispatch:
+    inputs:
+      git-ref:
+        description: Git Ref (Optional)    
+        required: false
+# Jobs
+jobs:
+  build-prod:
+      runs-on: ubuntu-latest
+      env:
+        CI: true
+        build_env: production
+      steps:
+      # Checkout the code
+        - uses: actions/checkout@v2
+      # Run Tests
+        - name: Run Unit Tests
+          uses: actions/setup-node@v1
+          with: 
+            node-version: 12.x
+        - run: |
+            yarn
+            yarn test --watchAll=false
+      # Build Code
+        - name: Build Static Site
```

Comment:
The production pipeline shouldn't have a build step. Both staging and production should share an artifact.